third_party/blink/renderer/platform/heap/member.h

```diff
@@ -133,8 +133,9 @@ struct MemberHashRecordReplayRegisteredPointerId
   static bool Equal(const U& a, const V& b) {
     return a == b;
   }
-};
 
+  static constexpr bool IsDeterministicHash = true;
```

Comment:
I think IsRecordReplayDeterministicHash would be better, this makes it easy to see this code is a recording related change we've made when browsing the source.

Reply:
That makes sense. Done.